swaybar/tray/tray.c

```diff
@@ -37,7 +38,7 @@ struct swaybar_tray *create_tray(struct swaybar *bar) {
 	sway_log(SWAY_DEBUG, "Initializing tray");
 
 	sd_bus *bus;
-	int ret = sd_bus_open_user(&bus);
+	int ret = sd_bus_open_user_with_description(&bus, "Swaybar tray bus");
```

Comment:
Since we have only a single `sd_bus *`, is it really useful to set the description?

Reply:
Nope. I was thinking dbus monitors might wanna know but that's just an assumption I made. Removed.